sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -52,6 +52,10 @@ public static Document documentFromObject(Object document, ObjectMapper mapper)
         return Document.FromObject(document, mapper);
     }
 
+    public static String toJsonString(Object document, ObjectMapper mapper) {
```

Comment:
Just for my understanding: Why is it modeled as string, isn't stream the right abstraction?

Reply:
where good question.

There are three options:
1. String
2. Byte[]
3. InputStream/OutputStream

Currently everywhere in SDK, String is passed to DocumentServiceRequest. The public constructors of DocumentServiceRequest only accept String. But DSR internally converts String to Byte[] and holds it as Byte array.

I think there should be a potential room for optimization by passing data as Byte[] to DSR, I am planning to look at this after this PR is merged (as this is not specific to document write).

the other option is InputStream and OutputStream. as mentioned above DSR holds data as byte[] array and support for InputStream/OutputStream might not currently be used anywhere or even may not be in working state. The other challenge is converting from InputStream/OuputStream to Flux which might require some work. I don't know yet where we are in that regard.

My plan is to study the perfs for these options after this PR and make the requires changes.